server/__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+const request = require('supertest');
+const db = require('../../db/connection');
+const testData = require('../../db/data/test-data/index');
+const seed = require('../../db/seeds/seed')
+const app = require('../app');
+
+afterAll(() => {
+    if (db.end) db.end();
+  });
+
+  beforeEach(() => seed(testData));
+
+  describe('1. GET /api/categories', () =>{
+
+    test('status:404, sent bad path', ()=>{
+        return request(app)
+        .get('/badPath')
+        .expect(404)
+        .then((response)=>{
+            expect(response.body.msg).toBe("Route not found");
+        })
+    })
+
```

Comment:
You can stick the above test in it's own describe block, it doesn't really link to any specific endpoint nor this one